packages/app/src/App.tsx

```diff
@@ -1,155 +1,126 @@
+import {ScaffolderPage, scaffolderPlugin} from '@backstage/plugin-scaffolder';
+import {orgPlugin} from '@backstage/plugin-org';
+import {SearchPage} from '@backstage/plugin-search';
+import {TechRadarPage} from '@backstage-community/plugin-tech-radar';
+import {DefaultTechDocsHome, TechDocsIndexPage, techdocsPlugin, TechDocsReaderPage,} from '@backstage/plugin-techdocs';
+import {TechDocsAddons} from '@backstage/plugin-techdocs-react';
+import {ReportIssue} from '@backstage/plugin-techdocs-module-addons-contrib';
+import {UserSettingsPage} from '@backstage/plugin-user-settings';
+import {apis} from './apis';
+import {entityPage} from './components/catalog/EntityPage';
+import {searchPage} from './components/search/SearchPage';
+import {Root} from './components/Root';
+import {HomepageCompositionRoot, VisitListener} from '@backstage/plugin-home';
+import {HomePage} from './components/home/HomePage';
+import {AlertDisplay, OAuthRequestDialog, SignInPage} from '@backstage/core-components';
+import {createApp} from '@backstage/app-defaults';
+import {AppRouter, FlatRoutes} from '@backstage/core-app-api';
+import {CatalogGraphPage} from '@backstage/plugin-catalog-graph';
+import {RequirePermission} from '@backstage/plugin-permission-react';
+import {catalogEntityCreatePermission} from '@backstage/plugin-catalog-common/alpha';
+import {ExplorePage} from '@backstage-community/plugin-explore';
+import {microsoftAuthApiRef} from "@backstage/core-plugin-api";
+import {LighthousePage} from '@backstage-community/plugin-lighthouse';
+import {DevToolsPage} from '@backstage/plugin-devtools';
+import {DaskOnboardingPage} from '@kartverket/backstage-plugin-dask-onboarding';
+import {pluginRiScNorwegianTranslation} from '@kartverket/backstage-plugin-risk-scorecard';
 import {OpencostPage} from "@kartverket/backstage-plugin-opencost";
 
 const app = createApp({
-  __experimentalTranslations: {
-    availableLanguages: ['en', 'no'],
-    resources: [pluginRiScNorwegianTranslation],
-  },
-  components: {
-    SignInPage: props => {
-      const configApi = useApi(configApiRef);
-      if (configApi.has('auth.providers.github') && configApi.getOptionalString('auth.environment') === 'development') {
-        return <SignInPage {...props} auto provider={{
-            id: 'github-auth-provider',
-            title: 'GitHub',
-            message: 'Sign in using GitHub',
-            apiRef: githubAuthApiRef,
-        }} />;
-      }
-
-      if (configApi.getOptionalString('auth.environment') != 'production') {
-        return <SignInPage {...props} auto providers={['guest']} />;
-      }
-        return <SignInPage {...props} auto provider={{
+    __experimentalTranslations: {
+        availableLanguages: ['en', 'no'],
+        resources: [pluginRiScNorwegianTranslation],
+    },
+    components: {
+        SignInPage: props => <SignInPage {...props} auto providers={['guest', {
```

Comment:
i dont think we should have the guest signinpage in prod?

Reply:
Good catch!😅